database/migrations/2019_04_22_043336_show_user.php

```diff
@@ -88,17 +91,21 @@ public function up()
 
                 -- Harusnya matiin kursor disini
 
-            -- info yang akan diberikan
-            SELECT
-                id, name, statusAktif, terakhirMeminjam, terakhirMemesan, hasReturned
-            FROM
-                users
-                inner join userDanPinjamanTerakhir on users.id = userDanPinjamanTerakhir.idUser
-                inner join userDanPemesananTerakhir on users.id = userDanPemesananTerakhir.idUser
-                inner join userDanStatusSudahDikembalikan on users.id = userDanStatusSudahDikembalikan.idUser;
+            insert into tempHasil (idUser,terakhirMeminjam,terakhirMemesan,hasReturned)
+                    select userDanPinjamanTerakhir.idUser,terakhirMeminjam,terakhirMemesan, hasReturned
+                    from userDanPinjamanTerakhir
+                        inner join userDanStatusSudahDikembalikan on userDanStatusSudahDikembalikan.idUser = userDanPinjamanTerakhir.idUser
+                        left outer join userDanPemesananTerakhir on userDanPemesananTerakhir.idUser = userDanPinjamanTerakhir.idUser;
```

Comment:
kenapa masih nge inner join ? should no need inner join anymore

Reply:
Okay lemme fix it